archesproject/arches
# 6551 resource instance workflow

### Types of changes


### Description of Change

This primarily fixes workflow steps where the step selects a resource instance to designate as the resourceid for the workflow; in such cases the widget is not provided with a graph when it loads and this breaks the step.

To Test:
- Load package: Arches-HER (or AFRH)
- Enter workflow: Communication
- Select an instance of Management Activity (if AFRH) or instance of Consultation (if Arches-HER)
- Proceed through workflow until end, then click "Jump to Resource" to ensure that the tiles were saved. to the originally selected resource instance.

### Issues Solved

#6551 

### Checklist


#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @ 

### Further comments
